apps/demo-game/src/services/SegmentService.ts

```diff
@@ -1,41 +1,31 @@
-import { Action } from '@gbl-uzh/platform'
+import { OutputFacts, PayloadSegment } from '@gbl-uzh/platform'
 import {
   computeScenarioOutcome,
   debugLog,
   diceRoll,
 } from '@gbl-uzh/platform/dist/lib/util'
-import { PeriodFacts, PeriodSegmentFacts } from '@graphql/index'
-import { PrismaClient } from '@prisma/client'
+import { produce } from 'immer'
 import * as R from 'ramda'
+import { PeriodFacts, PeriodSegmentFacts } from '../types/Period'
 
-export enum ActionTypes {
-  SEGMENT_INITIALIZE = 'SEGMENT_INITIALIZE',
+type SegmentFacts = {
+  diceRolls?: { bonds: number; stocks: number }[]
```

Comment:
diceRolls are not there in the input if we do a specific type for each function?